bootstrapper/internal/kubernetes/kubernetes.go

```diff
@@ -154,7 +153,7 @@ func (k *KubeWrapper) InitCluster(
 
 	k.log.Info("Prioritizing etcd I/O")
 	if err := k.etcdIOPrioritizer.PrioritizeIO(); err != nil {
-		return nil, fmt.Errorf("prioritizing etcd I/O: %w", err)
+		k.log.Warn("prioritizing etcd I/O", "error", err)
```

Comment:
```suggestion
		k.log.Warn("Prioritizing etcd I/O failed", "error", err)
```
Log messages should start uppercase